app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  before_save{email.downcase!}
```

Comment:
email.downcase! và email.downcase khác nhau gì em

Reply:
downcase : copy chuỗi hiện tại sang chuỗi mới in thường, chuỗi gốc sẽ không bị thay đổi giá trị
downcase! : trả về nil nếu giá trị trong chuỗi không thay đổi, chuỗi gốc sẽ bị thay đổi

Reply:
em tìm hiểu thêm nữa, vẫn thiếu :D 
mở `rails c` lên test nhé
siêng lên chị cho xem nhé